validator/keymanager/remote-web3signer/keymanager.go

```diff
@@ -113,166 +113,283 @@ func getSignRequestJson(ctx context.Context, validator *validator.Validate, requ
 	case *validatorpb.SignRequest_ContributionAndProof:
-		contributionAndProofRequest, err := web3signerv1.GetSyncCommitteeContributionAndProofSignRequest(request, genesisValidatorsRoot)
-		if err != nil {
-			return nil, err
-		}
-		if err = validator.StructCtx(ctx, contributionAndProofRequest); err != nil {
-			return nil, err
-		}
-		syncCommitteeContributionAndProofSignRequestsTotal.Inc()
-		return json.Marshal(contributionAndProofRequest)
+		return handleContributionAndProof(ctx, validator, request, genesisValidatorsRoot)
 	case *validatorpb.SignRequest_Registration:
-		validatorRegistrationRequest, err := web3signerv1.GetValidatorRegistrationSignRequest(request)
-		if err != nil {
-			return nil, err
-		}
-		if err = validator.StructCtx(ctx, validatorRegistrationRequest); err != nil {
-			return nil, err
-		}
-		validatorRegistrationSignRequestsTotal.Inc()
-		return json.Marshal(validatorRegistrationRequest)
+		return handleRegistration(ctx, validator, request)
+	case *validatorpb.SignRequest_Blob:
+		return handleBlob(ctx, validator, request, genesisValidatorsRoot)
+	case *validatorpb.SignRequest_BlindedBlob:
+		return handleBlindedBlob(ctx, validator, request, genesisValidatorsRoot)
 	default:
 		return nil, fmt.Errorf("web3signer sign request type %T not supported", request.Object)
 	}
 }
 
+func handleBlock(ctx context.Context, validator *validator.Validate, request *validatorpb.SignRequest, genesisValidatorsRoot []byte) ([]byte, error) {
+	bockSignRequest, err := web3signerv1.GetBlockSignRequest(request, genesisValidatorsRoot)
+	if err != nil {
+		return nil, err
+	}
+	if err = validator.StructCtx(ctx, bockSignRequest); err != nil {
+		return nil, err
+	}
+	blockSignRequestsTotal.Inc()
+	return json.Marshal(bockSignRequest)
+}
+
+func handleAttestationData(ctx context.Context, validator *validator.Validate, request *validatorpb.SignRequest, genesisValidatorsRoot []byte) ([]byte, error) {
+	attestationSignRequest, err := web3signerv1.GetAttestationSignRequest(request, genesisValidatorsRoot)
+	if err != nil {
+		return nil, err
+	}
+	if err = validator.StructCtx(ctx, attestationSignRequest); err != nil {
+		return nil, err
+	}
+	attestationSignRequestsTotal.Inc()
+	return json.Marshal(attestationSignRequest)
+}
+
+func handleAggregateAttestationAndProof(ctx context.Context, validator *validator.Validate, request *validatorpb.SignRequest, genesisValidatorsRoot []byte) ([]byte, error) {
+	aggregateAndProofSignRequest, err := web3signerv1.GetAggregateAndProofSignRequest(request, genesisValidatorsRoot)
+	if err != nil {
+		return nil, err
+	}
+	if err = validator.StructCtx(ctx, aggregateAndProofSignRequest); err != nil {
+		return nil, err
+	}
+	aggregateAndProofSignRequestsTotal.Inc()
+	return json.Marshal(aggregateAndProofSignRequest)
+}
+
+func handleSlot(ctx context.Context, validator *validator.Validate, request *validatorpb.SignRequest, genesisValidatorsRoot []byte) ([]byte, error) {
```

Comment:
A nit pick but this sounds less descriptive than the other handlers, should it be more like `handleAggregationSlot`